pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -1769,6 +1783,20 @@
     )
     private Set<String> brokerClientTlsProtocols = Sets.newTreeSet();
 
+    /**** --- Interceptor variables --- ****/
+
+    @FieldContext(
+        category = CATEGORY_INTERCEPTORS,
+        doc = "The directory to locate interceptors"
+    )
+    private String interceptorDirectory = "./interceptors";
```

Comment:
overall looks good to me. I see there are two sets of settings here, one is for `listeners` and the other one is for `interceptors`? I think we should just rename the listener interfaces to Interceptors. So it is consistent with client-side interceptor.

Reply:
Ok.